packages/aws-cdk-lib/aws-logs/lib/data-protection-policy.ts

```diff
@@ -0,0 +1,375 @@
+import { Stack } from '../../core';
+import { Construct } from 'constructs';
+/**
+ * Represents a data protection policy in a log group.
+ */
+export class DataProtectionPolicy {
+
+  /**
+   * Name of the data protection policy
+   *
+   * @default - 'data-protection-policy-cdk'
+   */
+  public readonly name: string;
+
+  /**
+   * Description of the data protection policy
+   *
+   * @default - 'cdk generated data protection policy'
+   */
+  public readonly description: string;
+
+  /**
+   * Version of the data protection policy
+   */
+  public readonly version: string;
+
+  /**
+   * Statements within the data protection policy. Must contain one Audit and one Redact statement
+   */
+  public statement: object;
+
+  constructor(scope: Construct, props: DataProtectionPolicyProps) {
+    this.name = props.name || 'data-protection-policy-cdk';
+    this.description = props.description || 'cdk generated data protection policy';
+    this.version = '2021-06-01';
+
+    var findingsDestination: FindingsDestination = {};
+
+    if (props.logGroupNameAuditDestination) {
+      findingsDestination.cloudWatchLogs = {
+        logGroup: props.logGroupNameAuditDestination,
+      };
+    }
+
+    if (props.s3BucketNameAuditDestination) {
+      findingsDestination.s3 = {
+        bucket: props.s3BucketNameAuditDestination,
+      };
+    }
+
+    if (props.deliveryStreamNameAuditDestination) {
+      findingsDestination.firehose = {
+        deliveryStream: props.deliveryStreamNameAuditDestination,
+      };
+    }
+
+    var identifierArns: string[] = [];
+    if (props.identifiers != null) {
+      for (let identifier of props.identifiers) {
+        let identifierArn = Stack.of(scope).formatArn({
+          resource: 'data-identifier',
+          region: '',
+          account: 'aws',
+          service: 'dataprotection',
+          resourceName: identifier.toString(),
+        });
+        identifierArns.push(identifierArn);
+      };
+    }
```

Comment:
When you change to the enum like class I would move this into the `toString()`
method

Reply:
`formatArn` depends on the scope, which would need to be passed into every DataIdentifier constructor. I think in this case going without that would be cleaner.

Reply:
fair enough